vendor/extensions/articles/db/seeds.rb

```diff
@@ -20,31 +20,33 @@
 
 Refinery::Articles::Article.destroy_all
 
-Refinery::Articles::Article.create(
+if Rails.env.downcase != "production"
```

Comment:
we probably don't. I was just being conservative since we're comparing a string.